CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changes
 
+## 2.0.0 (2022-05-04)
+
+### Final
```

Comment:
I think this title can be removed as there was no alpha-like version this time ;).

Reply:
Removed!